content/docs/getting-started/ocm-controllers/introduction.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Introduction
+description: "Learn about the OCM Controllers and their capabilities"
+icon: "🏁"
+weight: 10
```

Comment:
There is something wrong with the weights because Next points to About OCM Project instead of the next section in the ocm controller getting Started documentation.